other/react-native/jest.config.js

```diff
@@ -1,10 +1,11 @@
-const jestConfig = require('kcd-scripts/config').jest
+// const jestConfig = require('kcd-scripts/config').jest
 
-module.exports = Object.assign(jestConfig, {
+module.exports = {
   preset: 'react-native',
-  testEnvironment: 'node',
-  transformIgnorePatterns: ['<rootDir>/node_modules/(?!react-native)/'],
   rootDir: '../../',
   roots: ['.'],
+  transform: {
+    '^.+\\.js$': '<rootDir>/node_modules/react-native/jest/preprocessor.js',
+  },
```

Comment:
This is necessary because of https://github.com/facebook/react-native/issues/19859. Also there was issues when using the config from `kcd-scripts`, but I don't think we have to use that here correct?

Reply:
Yes, this is fine.

Thank you for figuring this out!